SU-HSDO/suhumsci
# ARCH-20 Stub out subtheme

# READY FOR REVIEW

# Summary
- Add remaining foundational files to subtheme

# Needed By (Date)
- End of sprint (5/9)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that foundation of theme is ready
3. The remaining items (SCSS components, etc.) will all come as part of future work
4. Are there any other foundational items that should be here now?

# Affected Projects or Products
- suhumsci
- su_humsci_theme

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-20

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
